### ISSUE TITLE
Cannot read property 'offsetParent' with seo

### ISSUE THREAD

**Issue by javialon26** [nuxt-community/nuxt-i18n#22]
Hello, excellent module :)

I have an error with seo option in true, the console shows the following error:

```Uncaught TypeError: Cannot read property 'offsetParent' of undefined at vue-meta.js:1080```

Dependencies:
Nuxt 1.3.0

**Comment by paulgv** [comment]
Hi @javialon26 !
Can you share some code to reproduce the issue? I'm not seing this error in my tests so it might be specific to your setup (config, modules installed etc.).
Thanks!

**Comment by javialon26** [comment]
After a few tests, the problem appears when the module @nuxtjs/auth is enabled. The version of the module is 4.0.0-rc.3.

**Comment by apmontgomery** [comment]
I'm having the same issue, but not using nuxtjs/auth.  It only occurs with seo set to true and following a nuxt link with switchLocalePath (i.e. not on initial page load).  The Error which seems to keep looping is the following: 
```
Uncaught TypeError: Cannot read property 'offsetParent' of undefined
    at vue-meta.js:1080
(anonymous) @ vue-meta.js:1080
setInterval (async)
destroyed @ vue-meta.js:1079
callHook @ vue.runtime.esm.js:2891
Vue.$destroy @ vue.runtime.esm.js:2698
removeVvals @ vval.js:123
patchChildren @ vval.js:143
beforeDestroy @ index.js:174
callHook @ vue.runtime.esm.js:2891
Vue.$destroy @ vue.runtime.esm.js:2673
removeVvals @ vval.js:123
patchChildren @ vval.js:143
beforeDestroy @ index.js:174
callHook @ vue.runtime.esm.js:2891
Vue.$destroy @ vue.runtime.esm.js:2673
beforeDestroy @ index.js:527
callHook @ vue.runtime.esm.js:2891
Vue.$destroy @ vue.runtime.esm.js:2673
destroy @ vue.runtime.esm.js:4106
invokeDestroyHook @ vue.runtime.esm.js:5643
patch @ vue.runtime.esm.js:5981
Vue.$destroy @ vue.runtime.esm.js:2696
destroy @ vue.runtime.esm.js:4106
invokeDestroyHook @ vue.runtime.esm.js:5643
invokeDestroyHook @ vue.runtime.esm.js:5648
invokeDestroyHook @ vue.runtime.esm.js:5648
invokeDestroyHook @ vue.runtime.esm.js:5648
patch @ vue.runtime.esm.js:5981
Vue.$destroy @ vue.runtime.esm.js:2696
destroy @ vue.runtime.esm.js:4106
invokeDestroyHook @ vue.runtime.esm.js:5643
removeVnodes @ vue.runtime.esm.js:5659
patch @ vue.runtime.esm.js:6072
Vue._update @ vue.runtime.esm.js:2643
updateComponent @ vue.runtime.esm.js:2761
get @ vue.runtime.esm.js:3111
run @ vue.runtime.esm.js:3188
flushSchedulerQueue @ vue.runtime.esm.js:2950
(anonymous) @ vue.runtime.esm.js:1809
flushCallbacks @ vue.runtime.esm.js:1730
Promise.then (async)
microTimerFunc @ vue.runtime.esm.js:1778
nextTick @ vue.runtime.esm.js:1822
queueWatcher @ vue.runtime.esm.js:3037
update @ vue.runtime.esm.js:3178
notify @ vue.runtime.esm.js:680
reactiveSetter @ vue.runtime.esm.js:1005
proxySetter @ vue.runtime.esm.js:3269
finish @ nuxt-loading.vue?b0c3:64
boundFn @ vue.runtime.esm.js:191
_callee3$ @ client.js:321
tryCatch @ runtime.js:62
invoke @ runtime.js:296
prototype.(anonymous function) @ runtime.js:114
step @ asyncToGenerator.js:17
(anonymous) @ asyncToGenerator.js:28
Promise.then (async)
step @ asyncToGenerator.js:27
(anonymous) @ asyncToGenerator.js:28
Promise.then (async)
step @ asyncToGenerator.js:27
(anonymous) @ asyncToGenerator.js:28
Promise.then (async)
step @ asyncToGenerator.js:27
(anonymous) @ asyncToGenerator.js:28
Promise.then (async)
step @ asyncToGenerator.js:27
(anonymous) @ asyncToGenerator.js:28
Promise.then (async)
step @ asyncToGenerator.js:27
(anonymous) @ asyncToGenerator.js:35
F @ _export.js:35
(anonymous) @ asyncToGenerator.js:14
render @ client.js:363
iterator @ vue-router.esm.js:1943
step @ vue-router.esm.js:1717
(anonymous) @ vue-router.esm.js:1718
(anonymous) @ vue-router.esm.js:1964
_callee$ @ client.js:54
tryCatch @ runtime.js:62
invoke @ runtime.js:296
prototype.(anonymous function) @ runtime.js:114
step @ asyncToGenerator.js:17
(anonymous) @ asyncToGenerator.js:28
Promise.then (async)
step @ asyncToGenerator.js:27
(anonymous) @ asyncToGenerator.js:35
F @ _export.js:35
(anonymous) @ asyncToGenerator.js:14
loadAsyncComponents @ client.js:78
iterator @ vue-router.esm.js:1943
step @ vue-router.esm.js:1717
step @ vue-router.esm.js:1721
step @ vue-router.esm.js:1721
runQueue @ vue-router.esm.js:1725
confirmTransition @ vue-router.esm.js:1972
transitionTo @ vue-router.esm.js:1874
push @ vue-router.esm.js:2181
push @ vue-router.esm.js:2534
handler @ vue-router.esm.js:444
invoker @ vue.runtime.esm.js:2000
fn._withTask.fn._withTask @ vue.runtime.esm.js:1798
15:35:50.999 
```

**Comment by apmontgomery** [comment]
maybe there is some way to not change the seo on the client?

**Comment by paulgv** [comment]
Hi @apmontgomery 
Any way you could share a sample project where I can reproduce this issue? Not sure what's going on there 😕

**Comment by frank-084** [comment]
Hi i've got this problem using vuelidate
it's appears to be a vue-meta problem, check there : 
https://github.com/declandewet/vue-meta/issues/187

**Comment by paulgv** [comment]
I just had the same problem, looks like this is caused by the `seo` feature. I'll work on a fix asap.

**Comment by mahmoudmy** [comment]
I have the same problem when enabling nuxt-auth module. Icreated a sample repo that produce this error:
https://github.com/mahmoudmy/nuxt-i18n-test
@paulgv

**Comment by paulgv** [comment]
The issue is fixed in vue-meta v1.4.4: https://github.com/declandewet/vue-meta/releases/tag/v1.4.4 🎉 
vue-meta still needs to be upgraded in Nuxt's dependencies but in the meantime, you can simply add vue-meta as a dependency to your own projects.
